src/commands/device/index.ts

```diff
@@ -81,6 +81,64 @@ export default class DeviceCmd extends Command {
 			params.uuid,
 			options.json
 				? {
+						$select: [
+							'id',
+							'belongs_to__application',
+							'belongs_to__user',
+							'actor',
+							'is_pinned_on__release',
+							'device_name',
+							'is_of__device_type',
+							'uuid',
+							'is_running__release',
+							'note',
+							'local_id',
+							'status',
+							'update_status',
+							'last_update_status_event',
+							'is_online',
+							'last_connectivity_event',
+							'is_connected_to_vpn',
+							'last_vpn_event',
+							'ip_address',
+							'mac_address',
+							'public_address',
+							'os_version',
+							'os_variant',
+							'supervisor_version',
+							'should_be_managed_by__release',
+							'should_be_operated_by__release',
+							'is_managed_by__service_instance',
+							'provisioning_progress',
+							'provisioning_state',
+							'download_progress',
+							'is_web_accessible',
+							'longitude',
+							'latitude',
+							'location',
+							'custom_longitude',
+							'custom_latitude',
+							'is_locked_until__date',
+							'is_accessible_by_support_until__date',
```

Comment:
@otaviojacobi I think that you need to remove this prop since it doesn't exist in oB, which I think will make this request to fail

Reply:
In reality I think that it would have been enough if we selected the same props in both cases (or at least have as much overlap as possible).